homeassistant/components/kaleidescape/media_player.py

```diff
@@ -0,0 +1,217 @@
+
+    async def async_turn_on(self) -> None:
+        """Send leave standby command."""
+        await self._device.leave_standby()
+
+    async def async_turn_off(self) -> None:
+        """Send enter standby command."""
+        await self._device.enter_standby()
+
+    async def async_media_pause(self) -> None:
+        """Send pause command."""
+        await self._device.pause()
+
+    async def async_media_play(self) -> None:
+        """Send play command."""
+        await self._device.play()
+
+    async def async_media_stop(self) -> None:
+        """Send stop command."""
+        await self._device.stop()
+
+    async def async_media_next_track(self):
+        """Send track next command."""
+        await self._device.next()
+
+    async def async_media_previous_track(self):
+        """Send track previous command."""
+        await self._device.previous()
+
+    @property
+    def available(self) -> bool:
+        """Return if device is available."""
+        return self._device.is_connected
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return device specific attributes."""
+        return DeviceInfo(
+            identifiers={(KALEIDESCAPE_DOMAIN, self._device.serial_number)},
+            name=self.name,
+            model=self._device.system.type,
+            manufacturer=KALEIDESCAPE_NAME,
+            sw_version=f"{self._device.system.kos_version}",
+            suggested_area="Theater",
+            configuration_url=f"http://{self._device.host}",
+        )
+
+    @property
+    def extra_state_attributes(self) -> dict:
+        """Return additional attributes about the state."""
+        return {
+            "media_location": self._device.automation.movie_location,
+            "play_status": self._device.movie.play_status,
+            "play_speed": self._device.movie.play_speed,
+            "video_mode": self._device.automation.video_mode,
+            "video_color_eotf": self._device.automation.video_color_eotf,
+            "video_color_space": self._device.automation.video_color_space,
+            "video_color_depth": self._device.automation.video_color_depth,
+            "video_color_sampling": self._device.automation.video_color_sampling,
+            "screen_mask_ratio": self._device.automation.screen_mask_ratio,
+            "screen_mask_top_trim_rel": self._device.automation.screen_mask_top_trim_rel,
+            "screen_mask_bottom_trim_rel": self._device.automation.screen_mask_bottom_trim_rel,
+            "screen_mask_conservative_ratio": self._device.automation.screen_mask_conservative_ratio,
+            "screen_mask_top_mask_abs": self._device.automation.screen_mask_top_mask_abs,
+            "screen_mask_bottom_mask_abs": self._device.automation.screen_mask_bottom_mask_abs,
+            "cinemascape_mask": self._device.automation.cinemascape_mask,
+            "cinemascape_mode": self._device.automation.cinemascape_mode,
+        }
```

Comment:
Generally we prefer these to be their own sensors on the `sensor` platform (for a followup PR)

Reply:
Removed.